<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TicketOrderDlg</class>
 <widget class="QDialog" name="TicketOrderDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ticket Order</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Customer:</string>
       </property>
       <property name="buddy">
        <cstring>customerLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="customerLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;When:</string>
       </property>
       <property name="buddy">
        <cstring>whenDateTimeEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateTimeEdit" name="whenDateTimeEdit">
       <property name="displayFormat">
        <string>dd/MM/yyyy HH:mm:ss</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&amp;Price:</string>
       </property>
       <property name="buddy">
        <cstring>priceSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="priceSpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="prefix">
        <string>$</string>
       </property>
       <property name="maximum">
        <double>5000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&amp;Quantity:</string>
       </property>
       <property name="buddy">
        <cstring>quantitySpinBox</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="quantitySpinBox">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Amount:</string>
       </property>
       <property name="buddy">
        <cstring>amountLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="amountLineEdit">
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TicketOrderDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TicketOrderDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
